test/features/submitButtonDisabled.test.tsx

```diff
@@ -0,0 +1,37 @@
+import faker from 'faker';
```

Comment:
no need for this file to be `.tsx`. We only need that extension if there is jsx on the page (</>). Please rename to `submitButtonDisabled.test.ts`